src/util.cc

```diff
@@ -166,5 +167,31 @@ Status SyncTitanManifest(Env* env, TitanStats* stats,
   return file->Sync(db_options->use_fsync);
 }
 
+Status CreateTitanManifest(Env* env, bool use_fsync,
```

Comment:
Can we reuse it with `BlobFileSet::OpenManifest`?

Reply:
It looks like `BlobFileSet::OpenManifest` has some overlap with `CreateTitanManifest`, but I'd like to have as little intrusion into Titan as possible

Reply:
If that, let's move it out of util.cc

Reply:
Ok, I moved it to titan_checkpoint_impl.cc